app/javascript/packs/PasswordHandler.js

```diff
@@ -0,0 +1,80 @@
+export const MIN_LENGTH = 10;
```

Comment:
Unter `../packs` liegen nur max. 2 Dateien, die den Gesamtpack für JS und CSS beschreiben und builden. 
Wenn du also neue JS Dateien hinzufügen willst, muss du das im eigenen Unterordner machen. Dann diese im `app/javascript/packs/application.js` einbinden, damit ein Pack gebaut wird.

Hier dazu mehr:
https://edgeguides.rubyonrails.org/webpacker.html#using-webpacker-for-javascript

Reply:
Der Javascript anteil wurde wie im Guide korrigiert.